lambdas/date_generator/handler.py

```diff
@@ -1,38 +1,32 @@
 import os
-from collections.abc import Callable, Sequence
+from collections.abc import Sequence
 from datetime import datetime, timedelta
 from itertools import product
-from typing import TypedDict, Unpack
-
-
-class HandlerKwargs(TypedDict, total=False):
-    platforms: Sequence[str]
-    now: Callable[[], datetime]
-    lookback_days: int
```

Comment:
Do you want to keep this, but rename to `Event` and use it as the type of the `event` param of `handler`?

Reply:
Updated! Should be good to rereview 🙇